pkgs/by-name/fo/forge-mtg/package.nix

```
@@ -6,31 +6,77 @@
   maven,
   makeWrapper,
   openjdk,
+  libGL,
+  makeDesktopItem,
+  imagemagick,
+  nix-update-script,
 }:
 
 let
-  version = "1.6.65";
+  version = "2.0.05";
 
   src = fetchFromGitHub {
     owner = "Card-Forge";
     repo = "forge";
     rev = "forge-${version}";
-    hash = "sha256-MCJl3nBHbX/O24bzD4aQ12eMWxYY2qJC5vomvtsIBek=";
+    hash = "sha256-71CZBI4FvN5X7peDjhv+0cdTYv8hWwzM8ePdvQSb6QI=";
   };
 
   # launch4j downloads and runs a native binary during the package phase.
   patches = [ ./no-launch4j.patch ];
 
 in
-maven.buildMavenPackage {
+maven.buildMavenPackage rec {
   pname = "forge-mtg";
   inherit version src patches;
 
-  mvnHash = "sha256-ouF0Ja3oGrlUCcT0PzI5i9FQ+oLdEhE/LvhJ0QGErvI=";
+  mvnHash = "sha256-krPOUaJTo5i3imkDvEkBJH3W01y1KypdvitqmZ5JMMA=";
 
   doCheck = false; # Needs a running Xorg
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [
+    makeWrapper
+    imagemagick
+  ];
+  desktopItem = makeDesktopItem {
```

```suggestion
  desktopItems = [ (makeDesktopItem {
```

and suffix the expression with `) ]`